Formula/makepkg.rb

```diff
@@ -31,23 +29,77 @@ class Makepkg < Formula
   uses_from_macos "m4" => :build
   uses_from_macos "libxslt"
 
+  on_macos do
+    depends_on "coreutils" => :test # for md5sum
+  end
+
+  patch :DATA
```

Comment:
This patch needs to be upstreamed.

Reply:
Definitely - I will add a link once I've submitted it to their Gitlab.

Reply:
I have submitted the patch to the mailing list and added a link to the mailing list archive.